crates/bevy_pbr/src/render/fog.wgsl

```diff
@@ -6,6 +6,7 @@
 // https://iquilezles.org/articles/fog/ (Atmospheric Fog and Scattering)
 
 fn scattering_adjusted_fog_color(
+    fog: Fog,
```

Comment:
Could the global `fog` uniform be renamed, so that in future changes to this code, we don't accidentally end up using the uniform instead of the argument?

Reply:
I ended up renaming the argument in the fog functions instead, to `fog_params`. Right now other uniforms have a 1:1 match between their binding name and type name, with only a difference in case, (e.g. `point_lights: PointLights` and `cluster_offsets_and_counts: ClusterOffsetsAndCounts`) so ideally I want to preserve that consistency.